client/src/App.js

```diff
@@ -1,24 +1,24 @@
-import React from 'react';
+import React, { Component } from 'react';
 import './App.css';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends Component {
+    componentDidMount() {
+        fetch("/api/v1/test-servlet")
+            .then((response) => response.text())
+            .then((text) => {
+                console.log("here is the text from servlet: ", text);
+            });
+    }
+
+    render() {
+        return (
+            <div className="App">
+                <div className="App-header">
+                    <h2>Welcome to React</h2>
+                </div>
```

Comment:
Maybe instead we can add the text from the server onto this view to verify its working.

```
constructor(props) {
  super(props);
  this.state = { text: "" };
}

componentDidMount() {
        fetch("/api/v1/test-servlet")
            .then((response) => response.text())
            .then((text) => {
                this.setState({text});
            });
    }

...

render() {
   return (<div className="App">
                <div className="App-header">
                    <h2>Welcome to React</h2>
                   {this.state.text}
                </div>
      </div>
   );
}
```

This way we can both verify the fetch is actually hitting the backend correctly and it'll be a good reference for others when we want to replace it with the actual clusters api.

We can also test the componentDidMount is working by verifying the text appears